stylelint/styles.css

```
@@ -0,0 +1,72 @@
+/* */
+
+.SiteHeader {
+    display: flex;
+    padding: 16px 16px 16px 16px;
+    color: #333;
+    color: #555;
+    background: #f5f5f5;
+}
+
+.some_class {
+    width: 100pixels;
```

Stylelint (17.9.0)

[error] Unknown unit "pixels" (unit-no-unknown)

(unit-no-unknown)